mmv1/products/firebase/terraform.yaml

```diff
@@ -87,6 +87,41 @@ overrides: !ruby/object:Overrides::ResourceOverrides
     custom_code: !ruby/object:Provider::Terraform::CustomCode
       custom_import: templates/terraform/custom_import/self_link_as_name.erb
       custom_delete: templates/terraform/custom_delete/firebase_app_deletion_policy.erb
+  AppleApp: !ruby/object:Overrides::Terraform::ResourceOverride
+    import_format: ["projects/{{project}}/iosApps/{{appId}}", "iosApps/{{appId}}", "{{appId}}"]
+    autogen_async: true
+    skip_sweeper: true # Skip sweeper generation and use hard-delete in hand-written sweeper
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "firebase_apple_app_basic"
+        min_version: "beta"
+        primary_resource_id: "default"
+        vars:
+          display_name: "Display Name Basic"
+          bundle_id: "apple.app."
+        test_env_vars:
+          org_id: :ORG_ID
+          project_id: :PROJECT_NAME
+      - !ruby/object:Provider::Terraform::Examples
+        name: "firebase_apple_app_full"
+        min_version: "beta"
+        primary_resource_id: "full"
+        vars:
+          display_name: "Display Name Full"
+          bundle_id: "apple.app."
+          app_store_id: "12345"
+          team_id: "9987654321" # Has to be a 10-digit number.
+        test_env_vars:
+          org_id: :ORG_ID
+          project_id: :PROJECT_NAME
+        test_vars_overrides:
```

Comment:
These overrides are no longer required since these vars are the same as in `vars:`

```suggestion
```

Reply:
per discussion - these are required to avoid a random suffix on these integer fields.